rest-service/manager_rest/storage/resource_models_base.py

```diff
@@ -137,3 +137,7 @@ def list_labels(labels):
                       created_at=label.created_at,
                       creator_id=label.creator.id)
                 for label in labels]
+
+    def check_unique_query(self):
+        if self.is_id_unique:
```

Comment:
we now have a good chance to get rid of `.is_id_unique` entirely - just implement this method in the relevant classes, and do `def check_unique_query(self): return`

Beause:
1. I'd rather we had only one thing related to uniqueness (this method) rather than two (method + flag)
2. I've always felt the flag is named backwards anyway :P